app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,112 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link    {color:#000;} 
+a:visited {color:#000;}  
+a:hover   {color:#000;}  
+a:active  {color:#000;}
+
+body {
+  background-color: #F0F0F0;
+  font-family: sans-serif;   
+}
+
+.login {
+  position: relative;
+  max-width: 250px;
+  background-color: white;
+  margin: 10px auto;
+  margin-top: 10%;
+  margin-bottom: 10px;
+  padding: 10px 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px` instead of `10px 10px`